### DISCUSSION TITLE
0.41

### DISCUSSION THREAD

**Discussion by astoilkov** [notaapp/talk#405] [Roadmap]
- Table improvements (done) — a lot of users have been requesting improvements to tables. It's not a new rendering (that would take us more time as it's hard) but we've improved the typing assistance around creating tables.
  - Auto-column resize — https://share.cleanshot.com/DZFt41CGYDTMlKJXvq1x
  - Hitting `Enter` will automatically create the entire next line. If the line is empty, it will delete the line and start a new paragraph.
  - Hitting `|` at the end of a table line will create the entire next line.
  - `Tab` and `Shift+Tab` now move through cells in the table.
- Search improvements (work in progress) — we'll be making search more capable and make it easier to find things. I will give more details soon. Thanks for the feedback around this @borekb.
- List improvements in the editor (done)
    - The editor will support code blocks inside of list items
    - Quickly transform list items — for example, in the line `1. |` where `|` is the cursor, typing `[]` will transform the item into a block task item.
- Text replacements (done) — we will support macOS native text replacements throughout the whole app. This resolves https://github.com/notaapp/talk/discussions/89 and https://github.com/notaapp/talk/discussions/387.
- Here are also the issues and improvements that have been reported and flagged with `done (unreleased` — [issues](https://github.com/notaapp/talk/issues?q=is%3Aopen+is%3Aissue+label%3A%22done+%28unreleased%29%22) and [discussions](https://github.com/notaapp/talk/discussions?discussions_q=is%3Aopen+label%3A%22done+%28unreleased%29%22).

I will update the list when we make progress or decide we will ship something new. We're experimenting with better tabs UX but we're not sure we'll ship it in the next release.

**Comment by GraphicOwls** [comment]
That's awesome! Love those table improvements! 

With checkboxes, does this mean the bullet will finally be removed in from of checkboxes?

**Reply by erusev** [reply]
> With checkboxes, does this mean the bullet will finally be removed in from of checkboxes?

We'll give it a try, but can't make any promises.

**Comment by rudolphmax** [comment]
I don't quite understand this post. Is this supposed to be shipped already? Or is this what's coming up in the next release? If so, any updates on when we can expect the features?

**Reply by astoilkov** [reply]
This is what's coming for the next release (no ETA yet). I just added the "planned" label on this. Do you think the label will work well?

**Comment by mourner** [comment]
It's now been exactly 8 months since the last release... Are you guys OK?

Also, honestly, coupled with the silence on the app maintenance, it's been extremely worrying to read about your plans on integrating AI into the app. I hope you can see past this craze of bloating everything with AI, and focus on what actually made us buy the app in the first place — simplicity and sharp focus on the basics.

**Reply by erusev** [reply]
Hi, and thanks for your concern. 

We are working on sth AI-related ([IntelliBar](https://intellibar.app/)), but it's separate product. We don't plan to add AI-related features to Nota. We also don't plan to abandon Nota. We still use Nota every day and want to make it better.

**Reply by mathewmitchell** [reply]
I don't want to rain on your parade but given that Apple is definitely adding AI-related assistance to their various OSes  is this really a prudent move on your part? Oh well.

**Reply by Arbiter313** [reply]
I have to agree, it's really hard not seeing Apple putting more AI tech into spotlight or Siri

**Reply by abhi010196** [reply]
Just pinging here to see if there are any updates planned before the end of this year?

**Reply by jayasio** [reply]
Just pinging here to see if there are any updates planned?
